securedrop/tests/test_journalist.py

```diff
@@ -3181,36 +3181,6 @@ def test_single_source_is_successfully_unstarred(journalist_app, test_journo, te
         assert not source.star.starred
 
 
-@flaky(rerun_filter=utils.flaky_filter_xfail)
```

Comment:
Accidental deletion?

Reply:
I replaced this test with a functional test for sessions timeout, `TestJournalistAppSessionTimeout` above in this PR, so that I could remove `config.SESSION_EXPIRATION_MINUTES = -1` (ie. code that mutates the config).

Reply:
My bad, I totally glossed through that I needed to review https://github.com/freedomofpress/securedrop/pull/6551 first.